WordPress/twentytwentyfive
# Fix: Remove duplicate sidebar template.

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/344

Follow-up of https://github.com/WordPress/twentytwentyfive/pull/347


`parts/sidebar.html` and `parts/right-aligned-sidebar.html` are the same. This PR removes the duplicate and keeps only `parts/sidebar.html`


**Screenshots**

N/A

**Testing Instructions**

1. Open the site editor.
2. Go to the `Pages` template.
3. Pick the `Page template for the right aligned blog` design.
4. Confirm that the sidebar is there, and it has the same design.
5. Go to the `Single Posts` template.
6. Pick the `Right-aligned single post` design.
7. Confirm that the sidebar is there, and it has the same design.
